cli/smartcontract/generate.go

```diff
@@ -35,12 +35,15 @@ var generatorFlags = []cli.Flag{
 }
 
 var generateWrapperCmd = cli.Command{
-	Name:        "generate-wrapper",
-	Usage:       "generate wrapper to use in other contracts",
-	UsageText:   "neo-go contract generate-wrapper --manifest <file.json> --out <file.go> --hash <hash> [--config <config>]",
-	Description: ``,
-	Action:      contractGenerateWrapper,
-	Flags:       generatorFlags,
+	Name:      "generate-wrapper",
+	Usage:     "generate wrapper to use in other contracts",
+	UsageText: "neo-go contract generate-wrapper --manifest <file.json> --out <file.go> [--hash <hash>] [--config <config>]",
+	Description: `Generates a Go wrapper for interacting with a smart contract. If the '--hash' flag
+	 is provided, the generated wrapper will include static calls optimized for that contract hash.
```

Comment:
s/`will include static calls optimized for that contract hash.`/`uses CALLT instructions for the target contract invocation as an optimization of the wrapper contract code.`